xml/System.Net.Http/HttpClient.xml

```diff
@@ -2686,10 +2686,31 @@ The <paramref name="requestUri" /> is not an absolute URI.
         <Parameter Name="request" Type="System.Net.Http.HttpRequestMessage" Index="0" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="request">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="request">The HTTP request message to send.</param>
+        <summary>Send an HTTP request.</summary>
```

Comment:
```suggestion
        <summary>Sends an HTTP request.</summary>
```

For each overloaded method, could you add some info to differentiate them? For example, they are displayed in an overloads table, like [here](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.send?view=net-5.0), and if they all say exactly the same thing, it's not helpful to the customer.

Reply:
I suggested the disambiguations below.